src/20_Components/amp-bind.html

```diff
@@ -0,0 +1,106 @@
+<!---{
+    "experiments": ["amp-bind"],
+    "preview": "default"
+}--->
+<!--
+  #### Introduction
+
+  [amp-bind](https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/amp-bind.md)
+  allows you to add custom interactivity to your pages beyond
+  using AMP's pre-built components.
+
+  It works by mutating elements in response to user actions via data binding
+  and JS-like expressions.
+
+  For example, amp-bind can be used to:
+  - Link two amp-carousel components into an image gallery with a scrolling
+    thumbnail list.
+  - Create an e-commerce product page where the UI changes if the user's
+    currently selected item is not available.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-bind/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+  <!-- #### Setup -->
+  <!--
+  Import the amp-bind component in the header.
+  -->
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+  <!-- #### Configuring initial state -->
+  <!--
+  amp-bind works by binding elements to local JSON "state". This state can be
+  initialized with one or more &lt;amp-state&gt; components.
```

Comment:
Absolutely. We actually discussed the dynamic pricing use case recently vis-a-vis `amp-fresh`. First want to figure out the best way to offer this while avoiding bad UX patterns e.g. significant page changes without any user action. Suggestions welcome! :)

/cc @ericlindley-g